DTS-STN/Life-Journeys
# Addressed a few identified bugs

# Description

[LJ-366 Wording for EI maternity in Quebec region](https://lj-decd.atlassian.net/browse/LJ-366?atlOrigin=eyJpIjoiOTg4ZjMyZDVlZTljNDA1NzlmMmIwZjk4NzM0Y2ZjMzgiLCJwIjoiaiJ9 )
[LJ-363 Text doesn't fit en francais](https://lj-decd.atlassian.net/browse/LJ-363?atlOrigin=eyJpIjoiNTI4NTk3Y2Y1Yzc0NDRhMmFjY2YwMGQ1NzA4ODUxNmMiLCJwIjoiaiJ9)
[LJ-364 replace "Greater Than Sign" with text one](https://lj-decd.atlassian.net/browse/LJ-364?atlOrigin=eyJpIjoiNWQ2YmEyZDgwNTU2NDYyYmJhYTRhMjM5ZGE3NDUyZjMiLCJwIjoiaiJ9)

Addressed a few small bugs in this PR. The first is changing the wording for the third Quebec link to reference QPP instead of EI. The second is changing the English translation for Quebec in the province selector to not have an `é`. The third change is to the report a problem component, so that the text doesn't bleed outside the button when shrinking the screen. Lastly, I changed the arrow in between the breadcrumbs to simply be a `>` instead of using the Font Awesome icon.

I noted in Slack that Alpha Site is currently using a chevron icon for their arrow and that it may be smart to follow suit since we'll users will be linking to our site from Alpha Site. For the time being I have switched to just using `>` as per our discussions and as per Design Canada

## Test Instructions

1. Pull in branch
2. Type `npm run dev`
3. Re-size screen to see that report a problem component scales properly now
4. Go to the `Having a Baby` page and select Quebec from the province drop down to see the new text for the third link


## Meets Definition of Done

- [x] Report a problem summary text doesn't bleed outside of the button
- [x] Quebec is properly spelled in the English province selector
- [x] The text for the third Quebec link references QPP instead of EI
